src/sentry/tasks/post_process.py

```diff
@@ -336,53 +363,53 @@ def post_process_group(
 
```

Comment:
how much of the above needs to be modularized, as well? Some of this stuff can return early so I didn't want to touch it

Reply:
That might make sense. A lot of this stuff is kind of a preamble and likely applies to all events, so for now maybe it's fine to leave it in place here.